Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -1683,7 +1683,6 @@ static ErrorOr<void> cmyk_to_rgb(JPEGLoadingContext& context)
         }
     }
 
-    context.cmyk_bitmap = nullptr;
```

Comment:
Eh, sure. I kind of liked not having two copies in memory, but I admit the economics aren't great and it's subtle.

Reply:
Not being able to call the main API is a real footgun IMO. And as you said, this will be soon a fallback and there is no need to optimize for fallbackers.